responsibleai/tests/test_rai_insights.py

```diff
@@ -351,15 +351,15 @@ def validate_rai_insights(
         assert len(rai_insights._category_dictionary) == \
             len(categorical_features)
         if len(categorical_features) == 0:
-            assert isinstance(rai_insights._string_ind_data, list)
+            assert isinstance(rai_insights._string_ind_data, np.ndarray)
         else:
             assert isinstance(rai_insights._string_ind_data, np.ndarray)
```

Comment:
This else condition could be removed.

Reply:
nice catch!  done!

Reply:
I refactored the code a bit more as well to remove some other duplications